code/modules/client/client procs.dm

```diff
@@ -429,3 +429,25 @@
 		var/obj/item/clothing/under/U = H.get_item_by_slot(slot_w_uniform)
 		if(istype(U))
 			U.update_holomap()
+
+/client/verb/SwapSides()
+	set name = "swapsides"
+	set hidden = 1
+	var/newsplit = 100 - text2num(winget(usr, "mainwindow.mainvsplit", "splitter"))
+	if(winget(usr, "mainwindow.mainvsplit", "right") == "rpane")
+		winset(usr, "mainwindow.mainvsplit", "right=mapwindow;left=rpane;splitter=[newsplit]")
+	else
+		winset(usr, "mainwindow.mainvsplit", "right=rpane;left=mapwindow;splitter=[newsplit]")
+
+/client/verb/modify_preferences(page as num)
+	set name = "modifypreferences"
+	set hidden = 1
+	if(!prefs.saveloaded)
+		to_chat(src, "<span class='warning'>Your character preferences have not yet loaded.</span>")
+		return
+	switch(page)
+		if(1)
```

Comment:
Alright fixed as much as I can, page itself is still the same though because it just depends on the argument passed by the skin which cant use a define anyway.